FE/qr-menu/src/components/Items.jsx

```diff
@@ -8,49 +8,69 @@ import {
-            <CardGroup key={subcategory}>
-              {items.map((item) => {
-                const { id, name, category, desc, price, allergens, img } =
-                  item;
-                return (
-                  <div
-                    key={id}
-                    className="card-container"
-                    style={{ margin: '20px' }}
-                  >
-                    <Card
-                      style={{
-                        width: '18rem'
-                      }}
-                    >
-                      <img alt="Sample" src={img} />
-                      <CardBody>
-                        <CardTitle tag="h5">{name}</CardTitle>
-                        <CardSubtitle className="mb-2 text-muted" tag="h6">
-                          {price}
-                        </CardSubtitle>
-                        <CardText>{desc}</CardText>
-                      </CardBody>
-                    </Card>
+      <div className="group-container">
+        <CardGroup className="card-group">
+          {menu.map((menuItem) => {
+            const { id, name, category, desc, price, allergens, img } =
+              menuItem;
+            return (
+              <div key={id} className="card-container">
+                <Card color="dark" inverse className="card">
+                  <div className="card-image">
+                    <img alt="Sample" src={img} />
                   </div>
-                );
-              })}
-            </CardGroup>
-          </div>
-        );
-      })}
+                  <CardBody>
+                    <CardTitle tag="h5">{name}</CardTitle>
+                    <CardSubtitle className="mb-2 text-muted" tag="h6">
+                      <span className="dot">
+                        <img src={Wheat} />
+                      </span>
+                      <span className="dot">
+                        <img src={Egg} />
+                      </span>
+                      <span className="dot">
+                        <img src={Milk} />
+                      </span>
+                      <span className="dot">
+                        <img src={Peanuts} />
+                      </span>
+                    </CardSubtitle>
+                    <CardText>{desc}</CardText>
+                    <div className="bottom-panel">
+                      <span className="price">€ {price}</span>
+                      <Button
+                        style={{ background: '#FFBE33' }}
+                        className="rounded-circle"
+                      >
+                        <img
+                          style={{ height: '30px', width: '30px' }}
+                          src={ShoppingCart}
+                          alt=""
+                        />
+                      </Button>
+                    </div>
+                  </CardBody>
+                </Card>
+              </div>
+            );
+          })}
+        </CardGroup>
+      </div>
+      );
```

Comment:
this shouldn't be here
![image](https://user-images.githubusercontent.com/79214528/204508551-f7d7c83f-bd28-40fc-ae43-5c691d3c8ad5.png)

Reply:
thanks, done